### PR TITLE
LOOM-1307 BpkPhoneInput Class 2 Overrides

### PR DESCRIPTION
**Pull request by jronald01**
**Class 2 Overrides**:

1. `BpkInput` and `BpkSelect` classname overrides are duplicated and therefore redundant.  Can still see classnames in each component after the local override removed.
2. `<div>` classname override added as a parent wrapper to each `BpkLabel`.

Remember to include the following changes:

- [x] Ensure the PR title includes the name of the component you are changing so it's clear in the release notes for consumers of the changes in the version e.g `[KOA-123][BpkButton] Updating the colour`
- [ ] `README.md` (If you have created a new component)
- [ ] Component `README.md`
- [ ] Tests
- [ ] Storybook examples created/updated
- [ ] Type declaration (`.d.ts`) files updated
- [ ] For breaking changes or deprecating components/properties, migration guides added to the description of the PR. If the guide has large changes, consider creating a new Markdown page inside the component's docs folder and link it here

### CODE REVIEW

File: packages/bpk-component-phone-input/src/BpkPhoneInput.js [hunk 1]
```diff
@@ -149,23 +149,18 @@ const BpkPhoneInput = (props: Props) => {
...
+          </BpkLabel>
+        </div>
         {/* $FlowFixMe[cannot-spread-inexact] - inexact rest. See 'decisions/flowfixme.md'. */}
         <BpkSelect
           {...commonProps}
           {...dialingCodeProps}
-          // TODO: className to be removed
-          // eslint-disable-next-line @skyscanner/rules/forbid-component-props
-          className={getClassName(dialingCodeProps.className)}
-          wrapperClassName={getClassName(dialingCodeProps.wrapperClassName)}
```
Comment (metalix2) [inline packages/bpk-component-phone-input/src/BpkPhoneInput.js:168]: So on closer inspection the `dialingCodeProps` set the className and wrapperClassName which means these ones here are overrides for the default props. But also we don't detect that were setting className here as it's being set from the spread from `dialingCodeProps` above. There may be more cases of this which might have been missed.

Comment (mungodewar) [reply packages/bpk-component-phone-input/src/BpkPhoneInput.js:168]: spread props should be caught by the _parent_ component so we are covered there (and if it's available on the public interface of the component there is nothing we can do either). 

"named" className props are potentially still problematic, however, there are far fewer of them and so I'm far less concerned.

Comment (jronald01) [reply packages/bpk-component-phone-input/src/BpkPhoneInput.js]: Ah, I reverted the `BpkInput` and `BpkSelect` changes and just left the `BpkLabel` ones in there as per the original Backpack audit doc.

Are we saying we are fine with the `BpkInput` and `BpkSelect` changes being included here?

Comment (mungodewar) [reply packages/bpk-component-phone-input/src/BpkPhoneInput.js]: Lets leave the PR as is, is there another ticket for the BpkInput? If not, let's create one to see if anything can be done about the remaining `className` there or we move it to allow.

Comment (jronald01) [reply packages/bpk-component-phone-input/src/BpkPhoneInput.js]: I'll create another ticket for it, cannot see anything currently that matches 👍

Comment (jronald01) [reply packages/bpk-component-phone-input/src/BpkPhoneInput.js]: New ticket created for `BpkInput`.

**Review by mungodewar — APPROVED** [review]

File: packages/bpk-component-phone-input/src/BpkPhoneInput.js [hunk 2]
```diff
@@ -178,28 +177,27 @@ const BpkPhoneInput = (props: Props) => {
...
+            {...rest}
+            id={id}
+            name={name}
+            value={displayValue}
+            type={INPUT_TYPES.tel}
+            onChange={handleChange}
+            // TODO: className to be removed
+            // eslint-disable-next-line @skyscanner/rules/forbid-component-props
+            className={getClassName(className)}
+          />
```
Comment (metalix2) [inline packages/bpk-component-phone-input/src/BpkPhoneInput.js:200]: This seems to be indented by 1 too many.

Comment (jronald01) [reply packages/bpk-component-phone-input/src/BpkPhoneInput.js]: Oh yeah, thanks for spotting Matt, fixed now.

**Pull request merged**